packages/botonic-react/src/index-types.ts

```diff
@@ -129,16 +129,29 @@ export interface WebchatAppArgs {
   visibility?: () => boolean
 }
 
+export enum SENDERS {
+  bot = 'bot',
+  user = 'user',
+  agent = 'agent',
+}
+
 export interface WebchatMessage {
   ack: 0 | 1
+  bolb: boolean
   buttons: ButtonProps[]
+  children: any
   data: any
   delay: number
   display: boolean
-  from: 'user' | 'bot'
+  enabletimestamps: boolean
```

Comment:
camelCase `enableTimestamps`?

Reply:
I'm not sure in the message.jsx it is written as 
`enabletimestamps = props.enabletimestamps || props.enableTimestamps,`

Reply:
I have looked up where this change comes from and it seems that the correct form is all lower case https://github.com/hubtype/botonic/pull/1063